sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobAsyncClient.java

```diff
@@ -638,34 +652,132 @@ Mono<EncryptedBlob> encryptBlob(Flux<ByteBuffer> plainTextFlux) {
         }
     }
 
+    private Flux<ByteBuffer> encryptV2(Flux<ByteBuffer> plainTextFlux, SecretKey aesKey) {
```

Comment:
For GA: Could encryptV1 and V2 from here and from policy be extracted to impl and shared?

Reply:
The policy should be decrypting, not encrypting. There might a few pieces I can extract out to share, but they're by and large the inverses of each other. Unless I'm misunderstanding something?

Reply:
Gotcha, they looked kind of similar. they're both quite large pieces of code. would it make sense to move both to util class in impl in long run (doesn't have to be preview)?

Reply:
Yea I added it to the list.